src/Views/ProgressView.vala

```diff
@@ -317,7 +317,12 @@ public class ProgressView : AbstractInstallerView {
         }
 
         new Thread<void*> (null, () => {
-            installer.install ((owned) disks, config);
+            if (Installer.App.test_mode) {
+                on_success ();
```

Comment:
Thinking more, you may need to put this inside an `Idle.add` due to it being in a thread.